src/openrc-run/openrc-run.c

```diff
@@ -363,7 +363,7 @@ svc_exec(const char *arg1, const char *arg2)
 	int flags = 0;
 	struct pollfd fd[2];
 	int s;
-	char *buffer;
+	char buffer[BUFSIZ];
```

Comment:
nit: Since this array doesn't have `static` linkage, I'd remove the word "static" from the commit message just to avoid confusion. Just "use an array instead of malloc" is fine.